source/javascripts/app/_remote_load.js.es6

```diff
@@ -0,0 +1,39 @@
+function updateToc(headers) {
+  let apiTitle = document.querySelector('[data-title="API"]');
+
+  let apiLinks = apiTitle.nextElementSibling;
+
+  if (!apiLinks) {
+    apiLinks = document.createElement('ul');
+    apiLinks.classList.add('toc-list-h2', 'active');
+    apiTitle.parentNode.appendChild(apiLinks);
+  }
+
+
+  apiLinks.innerHTML += headers.map((header) =>
+    `<li>
+        <a href="#${header.toLowerCase()}" class="toc-h2 toc-link" data-title="${header}">${header}</a>
+      </li>`
+  ).join('');
+}
+
+async function remoteLoad() {
+  let text = '<p style="color:red">Unable to load API docs</p>';
+
+  try {
+    const response = await fetch('https://s3-us-west-2.amazonaws.com/developer.grooveapp.com/production/public-api-documentation/docs.html')
+    if (response.status >= 400) {
```

Comment:
I think a more exacting check may be

```javascript
if (response.status === 200) {
  text = await response.text();
}
```

Reply:
If aws someday changed their status code to some 2xx that's not 200, but still included the response as expected, would we want to throw?

Reply:
@vijaypemmaraju it looks like you implemented this but just for completeness sake I'll address the comment!

If AWS ever did this it would a breaking change for their API and I assume we'd get a bit of a heads up.

Regardless, here's why I prefer the explicit `200` check - a `204` would not throw but then  the content would not be the default content, not to mention all the `300`s that would not throw that we'd potentially use the response text of.

Basically, we will only use the response text when it's a `200` response - the only response code we can bank on that should return the text we want. So then, if we know that, I don't see any downside to checking for that "bankable" response code directly.